templates/user/dashboard/guide.tmpl

```diff
@@ -0,0 +1,14 @@
+<div class="ui center tw-px-8 tw-py-8 guide">
+	{{svg "octicon-package" 24}}
+	<h3>{{ctx.Locale.Tr "home.guide_title"}}</h3>
+	<p>{{ctx.Locale.Tr "home.guide_desc"}}</p>
+	<div>
+		<a href="{{AppSubUrl}}/explore/repos">
```

Comment:
Maybe you need also to check whether `DISABLE_USERS_PAGE` is enabled.

Reply:
It has been checked. The logic here is actually equivalent to that of the navigation bar at the top.